### PR TITLE
PCHR-1380: HTML + CSS

### PR DESCRIPTION
**Pull request by BorisCompucorp**
**Commits:**
* Created call of modal on Record Sickness click. Added additional fields

**Screenshots:**

Click on "Record New Absence":

Click on "Record Sickness":

### CODE REVIEW

File: uk.co.compucorp.civicrm.hrleaveandabsences/views/shared/directives/leave-request-popup.html [hunk 1]
```diff
@@ -203,6 +203,37 @@ <h4 class="modal-title">Leave Request</h4>
                 </div>
               </div>
             </crm-loading>
+            <div class="row" ng-show="$ctrl.sickness == true">
```
Comment (deb1990) [inline uk.co.compucorp.civicrm.hrleaveandabsences/views/shared/directives/leave-request-popup.html]: Just `ng-show="$ctrl.sickness"` is fine

File: uk.co.compucorp.civicrm.hrleaveandabsences/views/shared/directives/leave-request-popup.html [hunk 2]
```diff
@@ -203,6 +203,37 @@ <h4 class="modal-title">Leave Request</h4>
...
+              <div class="col-xs-12 form-horizontal">
+              <hr>
+                <div class="form-group">
+                  <label class="col-xs-4 control-label">Reason<i class="fa fa-asterisk"></i></label>
+                  <div class="col-xs-8">
+                    <div class="chr_custom-select chr_custom-select--full">
+                      <select name="reason" class="ng-pristine ng-valid ng-not-empty ng-touched" id="">
```
Comment (deb1990) [inline uk.co.compucorp.civicrm.hrleaveandabsences/views/shared/directives/leave-request-popup.html]: remove `ng-pristine ng-valid ng-not-empty ng-touched" id=""`

File: uk.co.compucorp.civicrm.hrleaveandabsences/views/shared/directives/leave-request-popup.html [hunk 3]
```diff
@@ -203,6 +203,37 @@ <h4 class="modal-title">Leave Request</h4>
...
+                        <option value="- select -">- select -</option>
+                      </select>
+                    </div>
+                  </div>
+                </div>
+                <div class="form-group">
+                  <div class="col-xs-12">
+                    <label class="chr_custom-checkbox squared">
```
Comment (deb1990) [inline uk.co.compucorp.civicrm.hrleaveandabsences/views/shared/directives/leave-request-popup.html]: Remove any custom styles for checkbox and use default one, please remove styles from the theme PR also

File: uk.co.compucorp.civicrm.hrleaveandabsences/js/angular/src/leave-absences/shared/controllers/leave-request-popup-controller.js [hunk 4]
```diff
@@ -707,6 +709,10 @@ define([
       }
 
       /**
+       * Initialize record sickness fields
```
Comment (deb1990) [inline uk.co.compucorp.civicrm.hrleaveandabsences/js/angular/src/leave-absences/shared/controllers/leave-request-popup-controller.js]: These are not removed

File: uk.co.compucorp.civicrm.hrleaveandabsences/js/angular/src/leave-absences/shared/controllers/leave-request-popup-controller.js [hunk 5]
```diff
@@ -41,6 +41,8 @@ define([
       vm.leaveRequestDayTypes = [];
       vm.period = {};
       vm.statusLabel = '';
+      vm.sickness = false;
+      vm.sickness = directiveOptions.sickness;
```
Comment (deb1990) [inline uk.co.compucorp.civicrm.hrleaveandabsences/js/angular/src/leave-absences/shared/controllers/leave-request-popup-controller.js]: You are defining `vm.sickness` 2 times. Remove 1st one

File: uk.co.compucorp.civicrm.hrleaveandabsences/views/shared/directives/leave-request-popup.html [hunk 6]
```diff
@@ -203,6 +203,37 @@ <h4 class="modal-title">Leave Request</h4>
...
+                    <div class="chr_custom-select chr_custom-select--full">
+                      <select name="reason">
+                        <option value="- select -">- select -</option>
+                      </select>
+                    </div>
+                  </div>
+                </div>
+                <div class="form-group">
+                  <div class="col-xs-12">
+                    <label class="chr_custom-checkbox">
```
Comment (deb1990) [inline uk.co.compucorp.civicrm.hrleaveandabsences/views/shared/directives/leave-request-popup.html]: Do you really need `chr_custom-checkbox` class?

File: uk.co.compucorp.civicrm.hrleaveandabsences/js/angular/src/leave-absences/shared/controllers/leave-request-popup-controller.js (hunk 4 above)
Comment (BorisCompucorp) [inline uk.co.compucorp.civicrm.hrleaveandabsences/js/angular/src/leave-absences/shared/controllers/leave-request-popup-controller.js]: Oops

**Pull request merged**